pkg/constants/env_variables.go

```diff
@@ -20,7 +20,7 @@ const (
 	SuppressSkipSignatureVerificationWarning          = "TANZU_CLI_SUPPRESS_SKIP_SIGNATURE_VERIFICATION_WARNING"
 	CEIPOptInUserPromptAnswer                         = "TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER"
 	EULAPromptAnswer                                  = "TANZU_CLI_EULA_PROMPT_ANSWER"
-	E2ETestEnvironment                                = "TANZU_CLI_E2E_TEST_ENVIRONMENT"
+	E2ETestEnvironment                                = "TANZU_CLI_E2E_TEST_ENVIRONMENT" // environment variable to indicate that the CLI is running in E2E test environment
```

Comment:
nit: the comment could ne added on top the constant instead of adding at the end